allure/pytest_plugin.py

```diff
@@ -132,6 +133,14 @@ def attach(self, title, contents, attach_type):
                         type=attach_type)
         self.stack[-1].attachments.append(attach)
 
+    def dynamic_issue(self, *issues):
+        """
+        Attaches ``issues`` to the current active case
+        """
+        issues = [TestLabel(name=Label.ISSUE, value=issue) for issue in issues]
+        if self.test:
```

Comment:
How exactly can one use this function in order to this check be useful?
Also, let's move the list comprehension in to the `if` block.